docs/docs/06-tools/03-debugging.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Debugging
+id: Debugging
+description: Learn how to debug your Wing application
+keywords: [debugging, debug, test, vscode]
+---
+
+## Overview
+
+Internally Wing uses JavaScript to execute preflight and `inflight` code, so standard JavaScript debugging tools can be used to debug your Wing application. The best-supported debugger is the built-in VS Code one so this guide will focus on that. 
```

Comment:
I use `inflight` over "inflight" pretty often because it's a keyword. I would do the same with `const` or `class`. Should inflight specifically not use "`" or should other keywords also not get backticks?